config/crd/bases/networking.x-k8s.io_tlsroutes.yaml

```diff
@@ -10,13 +10,19 @@ metadata:
 spec:
   group: networking.x-k8s.io
   names:
+    categories:
+    - gateway-api
     kind: TLSRoute
     listKind: TLSRouteList
     plural: tlsroutes
     singular: tlsroute
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .metadata.creationTimestamp
```

Comment:
I recommend:
```
    - description: 'CreationTimestamp is a timestamp representing the server time
        when this object was created. It is not guaranteed to be set in happens-before
        order across separate operations. Clients may not set this value. It is represented
        in RFC3339 form and is in UTC. Populated by the system. Read-only. Null for
        lists. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#metadata'
      jsonPath: .metadata.creationTimestamp
      name: Age
      type: date
```

This will match Kubernetes core types. I also noticed when using Age it gives relative time - maybe that is a special kubectl feature?

Reply:
oh nevermind, probably its from `type`

Reply:
Woops, somehow I thought I was already setting type correctly. Thanks for catching this!